testing/btest/scripts/base/frameworks/telemetry/cluster.zeek

```diff
@@ -0,0 +1,81 @@
+# @TEST-DOC: Query the Prometheus endpoint on 9911 and smoke check that zeek_version_info{...} is contained in the response for all cluster nodes.
+#
+# @TEST-PORT: BROKER_PORT1
+# @TEST-PORT: BROKER_PORT2
+# @TEST-PORT: BROKER_PORT3
+# @TEST-PORT: BROKER_PORT4
+#
+# @TEST-REQUIRES: which curl
+# @TEST-EXEC: zeek --parse-only %INPUT
+# @TEST-EXEC: btest-bg-run manager-1 ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=manager-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-run logger-1 ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=logger-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-run proxy-1 ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=proxy-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-run worker-1  ZEEKPATH=$ZEEKPATH:.. CLUSTER_NODE=worker-1 zeek -b %INPUT
+# @TEST-EXEC: btest-bg-wait 10
+# @TEST-EXEC: btest-diff manager-1/.stdout
+
+@load base/frameworks/cluster
+@load base/frameworks/telemetry
+@load base/utils/active-http
+
+# Query the Prometheus endpoint using ActiveHTTP for testing, oh my.
```

Comment:
An aside: when these test setups start to feel hairy, we could instead put them into the cluster management testsuite — `docker-compose` and `curl` are arguably more straightforward than `btest-bg-run` and `ActiveHTTP`, and there's no need to keep the testsuite strictly focused on the Management framework. (Seems fine for me here, but something to keep in mind for the future — we could no doubt benefit from more cluster framework testing.)